docs/ossec_alerts.md

```diff
@@ -40,7 +41,6 @@ These are the values you must specify in `prod-specific.yml`:
  * Email username to authenticate to the SMTP relay: `sasl_username`
  * Domain name of the email used to send OSSEC alerts: `sasl_domain`
  * Password of the email used to send OSSEC alerts: `sasl_password`
- * The fingerprint of your SMTP relay: `smtp_relay_fingerprint`
```

Comment:
Since we know that `secure` will work with gmail and the `verify` option provides less protections for forged DNS requests, I do not think it is necessary for us to support the `verify` option.

Think our stance should be is that whatever smtp relay they pick, it needs to have the certificates provisioned correctly to pass the stricter `secure` validation or they will need to manage keeping the fingerprint value updated.